spec/forms/schools/cohorts/setup_wizard_spec.rb

```diff
@@ -55,22 +56,23 @@
       allow(wizard).to receive(:what_changes).and_return(what_changes)
     end
 
-    context "when the SIT does not expect any ECTs" do
-      it "does not send the pilot survey" do
-        expect(Induction::SetCohortInductionProgramme)
-          .to receive(:call).with(programme_choice: :no_early_career_teachers,
-                                  school_cohort:,
-                                  opt_out_of_updates: true,
-                                  delivery_partner_to_be_confirmed: false)
-        expect(Induction::SetSchoolCohortAppropriateBody)
-          .to receive(:call).with(school_cohort:,
-                                  appropriate_body_id: 3,
-                                  appropriate_body_appointed: true)
-        expect {
-          wizard.success
-        }.not_to have_enqueued_mail(SchoolMailer, :cohortless_pilot_2023_survey_email)
-      end
-    end
+    # FIXME: removing while decision is made whether to keep/update/remove the survey for 2024
+    # context "when the SIT does not expect any ECTs" do
+    #   it "does not send the pilot survey" do
+    #     expect(Induction::SetCohortInductionProgramme)
+    #       .to receive(:call).with(programme_choice: :no_early_career_teachers,
+    #                               school_cohort:,
+    #                               opt_out_of_updates: true,
+    #                               delivery_partner_to_be_confirmed: false)
+    #     expect(Induction::SetSchoolCohortAppropriateBody)
+    #       .to receive(:call).with(school_cohort:,
+    #                               appropriate_body_id: 3,
+    #                               appropriate_body_appointed: true)
+    #     expect {
+    #       wizard.success
+    #     }.not_to have_enqueued_mail(SchoolMailer, :cohortless_pilot_2023_survey_email)
+    #   end
+    # end
```

Comment:
`xcontext` here makes it clear in the test output that this is temporarily disabled.

```suggestion
    xcontext "when the SIT does not expect any ECTs" do
      it "does not send the pilot survey" do
        expect(Induction::SetCohortInductionProgramme)
          .to receive(:call).with(programme_choice: :no_early_career_teachers,
                                  school_cohort:,
                                  opt_out_of_updates: true,
                                  delivery_partner_to_be_confirmed: false)
        expect(Induction::SetSchoolCohortAppropriateBody)
          .to receive(:call).with(school_cohort:,
                                  appropriate_body_id: 3,
                                  appropriate_body_appointed: true)
        expect {
          wizard.success
        }.not_to have_enqueued_mail(SchoolMailer, :cohortless_pilot_2023_survey_email)
      end
    end
```

Reply:
Good idea 👍